src/map/guild.cpp

```diff
@@ -802,7 +802,7 @@ void guild_member_joined(struct map_session_data *sd) {
 		guild_request_info(sd->status.guild_id);
 		return;
 	}
-	if (strcmp(sd->status.name,g->master) == 0) {	// set the Guild Master flag
+	if (sd->state.connect_new == 1 && strcmp(sd->status.name,g->master) == 0) {	// set the Guild Master flag
```

Comment:
This is probably related to the problem with the clan counter, because this function is called too often now.

Reply:
See #5204

Reply:
Uhm... this bug return!!!

Reply:
Or more specifically, it seems that this fix was not applied. At least the change history in guild.cpp says it was never implemented.